src/pages/Register.ts

```diff
@@ -242,7 +242,7 @@ const Register = () => {
       ]),
       div({ style: { marginTop: '3rem' } }, [
         h(ButtonPrimary, { disabled: errors || busy, onClick: register }, ['Register']),
-        h(ButtonSecondary, { style: { marginLeft: '1rem' }, onClick: signOut }, ['Cancel']),
+        h(ButtonSecondary, { style: { marginLeft: '1rem' }, onClick: () => signOut('requested') }, ['Cancel']),
```

Comment:
This was calling `signOut` with a mouse event.

Reply:
@sjkobori FYI. This looks like it would previously send sign out metrics with the cause as "unspecified".